src/main/java/org/opensearch/sdk/ExtensionSettings.java

```diff
@@ -120,20 +130,25 @@ public ExtensionSettings(String extensionName, String hostAddress, String hostPo
      */
     public ExtensionSettings(
         String extensionName,
+        String shortName,
         String hostAddress,
         String hostPort,
         String opensearchAddress,
         String opensearchPort,
         Map<String, String> securitySettings
     ) {
-        this(extensionName, hostAddress, hostPort, opensearchAddress, opensearchPort);
+        this(extensionName, shortName, hostAddress, hostPort, opensearchAddress, opensearchPort);
         this.securitySettings = securitySettings;
     }
 
     public String getExtensionName() {
         return extensionName;
     }
 
+    public String getShortName() {
+        return shortName;
+    }
```

Comment:
This getter should have a test in `TestExtensionSettings.java`

Reply:
This has been added to a few different tests to verify its behavior